inc/admin/class-envato-market-admin.php

```diff
@@ -960,8 +963,11 @@ function ( $val ) {
 		 * @since 1.0.0
 		 */
 		public function authorize_token_permissions() {
+			if (wp_get_environment_type() === 'development') {
```

Comment:
It might be better to use our own custom env_var instead of checking for the environment, so that we don't break the plugin for other Wordpress developers.